api/controllers/comments-controllers.js

```diff
@@ -0,0 +1,16 @@
+const { removeCommentById, checkCommentsIdExists } = require("../models/comments-models");
+
+exports.deleteCommentById = (req, res, next) => {
+    const { comment_id } = req.params;
+    const deleteCommentPromises = [removeCommentById(comment_id)];
+
+    if(comment_id){
+      deleteCommentPromises.push(checkCommentsIdExists(comment_id));
```

Comment:
Just a side note, but will there ever not be a comment_id? You could probably just do this in the promise.all to begin with, without doing this push stuff..

Just a possible refactor